pkg/virt-config/config_test.go

```diff
@@ -141,14 +121,16 @@ var _ = Describe("ConfigMap", func() {
 		table.Entry("is invalid, GetNodeSelectors should return the default", "-1", nil),
 	)
 
-	table.DescribeTable(" when machineType", func(value string, result string) {
-		clusterConfig, _, _, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
-			Data: map[string]string{virtconfig.MachineTypeKey: value},
-		})
+	table.DescribeTable(" when machineType", func(cpuArch string, machineType string, result string) {
+		clusterConfig, _, _, _ := testutils.NewFakeClusterConfigWithCPUArch(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.MachineTypeKey: machineType},
+		}, cpuArch)
 		Expect(clusterConfig.GetMachineType()).To(Equal(result))
 	},
-		table.Entry("when set, GetMachineType should return the value", "pc-q35-3.0", "pc-q35-3.0"),
-		table.Entry("when unset, GetMachineType should return the default", "", virtconfig.DefaultMachineType),
+		table.Entry("when set, GetMachineType should return the value", "", "pc-q35-3.0", "pc-q35-3.0"),
+		table.Entry("when unset, GetMachineType should return the default", "amd64", "", virtconfig.DefaultAMD64MachineType),
```

Comment:
These three tests all have the same test description, that is why the unit tests fail on

```
{"component":"","level":"fatal","msg":"Could not merge JUnit files","pos":"junit-merger.go:40","reason":"test 'ConfigMap  when machineType when unset, GetMachineType should return the default' was executed multiple times","timestamp":"2021-06-30T08:47:22.440889Z"}
```

Just add e.g. `with amd64`, `with arm64` and so forth to the entry description.

Reply:
> ,"level":"fatal","msg":"Could not merge JUnit files","pos":"junit-merger.go:40"

Ok, I see.